views/partials/navbar.hbs

```diff
@@ -37,7 +37,7 @@
           <a href="https://account.codingblocks.com" class="nav-link">😎 Profile</a>
         </li>
         <li class="nav-item">
-          <a href="https://account.codingblocks.com" class="nav-link">🔓 Logout</a>
```

Comment:
this should be a separate change as well, as it is unrelated to hit counter and pagination

Reply:
I have included this separately under your authorship 
https://github.com/coding-blocks/shortlr2-backend/commit/f9fa9fff63f2e51ce178d367655cde1f8aa36b9d